packages/itwinui-css/backstop/tests/assets/theme.js

```diff
@@ -120,16 +151,32 @@ class ThemeButton extends HTMLElement {
     `;
   };
 
+  changeBackground = ({ target: { value: _background } }) => {
+    if (_background == 'bg1') {
```

Comment:
Ah, sorry, can you make this `===` instead of `==`? It's more type safe - the code won't change the type to make comparisons. It doesn't really matter in a lot of circumstances, but it's best practice.

Reply:
Changed to:
```js
if (_background === 'bg1') {
```